linux_os/guide/services/xwindows/disabling_xwindows/package_xorg-x11-server-common_removed/ansible/shared.yml

```diff
@@ -0,0 +1,18 @@
+# platform = multi_platform_sle,multi_platform_rhv,multi_platform_fedora,Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,Oracle Linux 7,Oracle Linux 8
+# reboot = true
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Ensure xorg-x11-server-common is removed
+  package:
+    name: "xorg-x11-server-common"
+    state: absent
+
+- name: get default target
+  command: systemctl get-default
+  register: default_target
+
+- name: Switch to multi-user runlevel
+  command: systemctl set-default multi-user.target
+  when: default_target.stdout != "multi-user.target"
```

Comment:
NACK. This appears to do much more than remove a package, which is all this check should do.

Reply:
reverted